manifests/init.pp

```diff
@@ -36,6 +36,8 @@
 # $enable_ostree::        Boolean to enable ostree plugin. This requires existence of an ostree install.
 #                         type:boolean
 #
+# $github_username::      Github username to add remotes for
+#
```

Comment:
I was thinking about this the other day, if people give the git repo name (foreman-docker) to puppet, just replace the - with a _ in the gem name since *those* are consistent.

I get the point about enabling particular plugins but it's a lot to maintain especially when stuff is popping up all the time.